packages/cli/assets/pluginPackage/src/modelDefine.ts

```diff
@@ -0,0 +1,23 @@
+import {
+  ModelDefineType,
+  UniDataset,
+  ModelDefineArgsType,
+  UniModel,
+  Sample
+} from '@pipcook/pipcook-core';
+
+const templateModelDefine: ModelDefineType = async (data: UniDataset, args: ModelDefineArgsType): Promise <UniModel> => {
+
+  const model = {}; // Some model defination
+
+  const retModelDefine: UniModel = {
+    model,
+    predict: async function (input: Sample) {
```

Comment:
Ditto, remove `async`, or
```js
const pred = await this.model.predict(input.data);
```

Reply:
Not sure if model.predict is an async function since this is binding from Python. Any references here?

Reply:
So remove `async` is more strict.

Reply:
So remove async is more strict.

Reply:
OK, I checked the `ModelDefineType`. You are correct.